### PR TITLE
Problem answer distribution will only return the answer_value.

### PR DESCRIPTION
**Pull request by dsjen**
@jbau @rocha @clintonb

### CODE REVIEW

**Comment by jbau** [comment]
:+1:

**Comment by clintonb** [comment]
:+1:

File: analytics_data_api/v0/views/problems.py [hunk 1]
```diff
@@ -30,9 +30,7 @@ class ProblemResponseAnswerDistributionView(generics.ListAPIView):
             * count: The number of times the answer in this collection was
               given.
             * value_id: The ID of the answer in this collection.
-            * answer_value_text: The text of this answer, for text problems.
-            * answer_value_numeric: The number for this answer, for numeric
-              problems.
+            * answer_value: An answer for this problems.
```
Comment (brianhw) [inline analytics_data_api/v0/views/problems.py]: s/problems/problem

Comment (brianhw) [inline analytics_data_api/v0/views/problems.py]: Or use the description for answer_value_text.

Comment (dsjen) [inline analytics_data_api/v0/views/problems.py]: Good catch--updated to "problem."

**Pull request merged**